Resources/Catalog/RucioFileCatalogClient.py

```diff
@@ -0,0 +1,545 @@
+from __future__ import print_function
```

Comment:
Add a docstring

Reply:
And in general all docstrings in this file also need variable explanations e.g.
```
:param x: is a good variable
```